app/main/menu/submenu-items.js

```diff
@@ -140,8 +141,11 @@ items.reloadApp = {
   click(item, focusedWindow) {
     if (focusedWindow) {
       focusedWindow.webContents.send('reload-window');
-      focusedWindow.webContents.once('did-finish-load', () => {
-        focusedWindow.webContents.send('new-tab');
+
+      electron.ipcMain.once('guest-window-id', (event, id) => {
```

Comment:
Is this right?  If `guest-window-id` is coming from Electron (I think it is, 'cuz I don't see it in Tofino), are we ceding ID generation to Electron?  Do we expect Electron's ID generation to have good properties?  (I.e., why isn't this just 1, 2, 3... each time we restart?)